packages/manager/apps/dedicated/client/app/account/user/users/sso/details/user-users-sso-details.html

```diff
@@ -0,0 +1,60 @@
+<oui-header heading="{{:: 'user_users_sso_detail_title' | translate }}">
+</oui-header>
+<div>
+    <a
+        class="col-md-12 mb-3"
+        data-ng-href="{{'#/useraccount/users' }}"
+        title="{{ 'user_users_sso_detail_back' | translate }}"
```

Comment:
i am not sure this attribute is part of `A` HTML element ?

Reply:
Indeed ! Removed.